### PR TITLE
PWN-8589 - Add fee to total amount if user pays for claim

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-8589

## Description of Work

@kabramovp2p approved

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt [hunk 1]
```diff
@@ -18,8 +19,10 @@ data class BridgeFee(
     val name: String,
     val decimals: Int
 ) : Parcelable {
-    val amountInToken
-        get() = amount?.toBigDecimal()?.orZero()?.divide(decimals.toPowerValue()).orZero()
+    val amountInToken: BigDecimal
+        get() = amount?.toBigDecimal()
+            .orZero()
```
Comment (eduardmaximovich) [inline app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt:24]: zero division

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt:24]: you forgot to transform from lamports. 

`amount.toBigIntegerOrZero().fromLamports(decimals)`

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt:24]: sorry, not my code, won't touch the division and stuff

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt]: reverted

**Review by Davran331122 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt [hunk 2]
```diff
@@ -18,8 +19,8 @@ data class BridgeFee(
     val name: String,
     val decimals: Int
 ) : Parcelable {
-    val amountInToken
-        get() = amount?.toBigDecimal()?.orZero()?.divide(decimals.toPowerValue()).orZero()
+    val amountInToken: BigDecimal
+        get() = amount?.toBigDecimal()?.let { it.divide(decimals.toPowerValue()) }.orZero()
```
Comment (eduardmaximovich) [inline app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt:23]: I know this isn't part of the code you worked on, but decimal division without rounding and specified scaling might cause either `ArithmeticException` or incorrect calculations. 
```suggestion
        get() = amount.toBigDecimalOrZero().divideSafe(decimals.toPowerValue())
```

**Review by eduardmaximovich — APPROVED** [review]

**Pull request merged**